met-web/src/components/engagement/form/AddSurveyBlockTabs.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import TabContext from '@mui/lab/TabContext';
+import RichTextEditor from './RichTextEditor';
+import { MetTab, MetTabList, MetTabPanel } from './StyledTabComponents';
+import { upcomingText, openText, closedText } from 'constants/submissionStatusText';
+
+const AddSurveyBlockTabs = () => {
+    const [value, setValue] = React.useState('upcoming');
+
+    return (
+        <Box sx={{ width: '100%', typography: 'body1' }}>
+            <TabContext value={value}>
+                <Box sx={{ marginBottom: '0.25em' }}>
+                    <MetTabList
+                        onChange={(_event: React.SyntheticEvent, newValue: string) => setValue(newValue)}
+                        TabIndicatorProps={{
+                            style: { transition: 'none', display: 'none' },
+                        }}
+                    >
+                        <MetTab label="Upcoming" value="upcoming" />
```

Comment:
one last thing, you can perhaps create a constants file in the same folder and just add an object like 
![image](https://user-images.githubusercontent.com/91914654/208951648-29ba1bee-fd5b-4b82-9754-736477692315.png)